actions-server/.env

```diff
@@ -1,2 +1,2 @@
-HASURA_ENDPOINT=http://localhost:8080/v1/graphql
+HASURA_ENDPOINT=http://graphql-engine:8080/v1/graphql
```

Comment:
I'd like this to be localhost:8000 still.  if it can work for `app` FE to talk to gql BE, it should work here too..

Reply:
if left as localhost atm

```
actions-server    | This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
actions-server    | FetchError: request to http://localhost:8080/v1/graphql failed, reason: connect ECONNREFUSED 127.0.0.1:8080
actions-server    |     at ClientRequest.<anonymous> (/usr/app/node_modules/node-fetch/lib/index.js:1455:11)
actions-server    |     at ClientRequest.emit (events.js:322:22)
actions-server    |     at ClientRequest.EventEmitter.emit (domain.js:482:12)
actions-server    |     at Socket.socketErrorListener (_http_client.js:426:9)
actions-server    |     at Socket.emit (events.js:310:20)
actions-server    |     at Socket.EventEmitter.emit (domain.js:482:12)
actions-server    |     at emitErrorNT (internal/streams/destroy.js:92:8)
actions-server    |     at emitErrorAndCloseNT (internal/streams/destroy.js:60:3)
actions-server    |     at processTicksAndRejections (internal/process/task_queues.js:84:21)
```

Reply:
likely this - https://stackoverflow.com/questions/39046413/docker-http-requests-between-containers

Reply:
`host.docker.internal` also works. 

but won't work for `yarn gql-gen` in actions-server directory.

Reply:
actually maybe this is less relevant, since to gql gen anyways... 

you'll need to run something like this:

```
docker-compose exec actions-server yarn gql-gen
docker-compose exec app yarn gql-gen
```

Reply:
^^^ it's about whether you are running within a docker container or not (localhost will mean different things)